README.md

```diff
@@ -18,6 +18,47 @@ This project is a template for [React Native](https://reactnative.dev/) that can
 
 NodeJS (version 16 or later) and NPM is required.
 To make sure you have them available on your machine, try running the following command
+## About Us
+
+Welcome to Awesome App! We're a small team of developers passionate about creating library that make people's lives easier.
```

Comment:
How about `Welcome to Awesome App! We're a small team of developers passionate at making our own developer lives easier.`